characterize.py

```diff
@@ -353,18 +340,31 @@ def execute(self, request: ServiceRequest) -> None:
 
             filename_extracted = bp or rp or t or nn
             if filename_extracted.rsplit("\\")[-1].strip():
-                lnk_result_section.add_tag(tag_type="file.name.extracted", value=filename_extracted.rsplit("\\")[-1])
+                lnk_result_section.add_tag("file.name.extracted", filename_extracted.rsplit("\\")[-1])
             elif extra_targets:
                 heur = Heuristic(7)
                 heur_section = ResultKeyValueSection(heur.name, heuristic=heur, parent=lnk_result_section)
                 for k, v in extra_targets.items():
                     filename_extracted = v
                     heur_section.set_item(k, v)
-                    heur_section.add_tag(tag_type="file.name.extracted", value=v.rsplit("\\")[-1])
+                    heur_section.add_tag("file.name.extracted", v.rsplit("\\")[-1])
+
+            if "icon_location" in features["data"]:
+                deceptive_icons = ["wordpad.exe", "shell32.dll", "explorer.exe", "msedge.exe"]
```

Comment:
I've also seen the icon for Word being used, maybe this could be added?

Reply:
Usually, a real word shortcut file as an icon named "wordicon.exe" and the target executable is "winword.exe".
Do you see malicious samples using wordicon.exe or winword.exe?
I searched for examples, and a quick search didn't bring up any.

Reply:
I was confused about something else. Conversation resolved!